JITpackage/Source/grb_jitpackage.c

```diff
@@ -112,14 +113,61 @@ bool match_prefix (char *string, char *prefix)
 int main (int argc, char **argv)
 {
 
+    //--------------------------------------------------------------------------
+    // get list of files to be processed
+    //--------------------------------------------------------------------------
+
+    char **file_list = NULL;
+    size_t nfiles = 0;
+
+    if (argc == 2 && argv[1][0] == '@')
+    {
+        // input argument is a "response file" containing the file list
+
+        // open file
+        FILE *fr = fopen (argv[1]+1, "rb") ;
+        OK (fr != NULL) ;
+
+        // get number of lines in file
+        char ch;
+        do
+        {
+            ch = fgetc (fr);
+            if (ch == '\n')
+                nfiles++;
+        } while (ch != EOF);
+
+        // read file list from response file
+        rewind (fr);
+        file_list = malloc ( (nfiles+1) * sizeof (file_list) );
+        // prepend empty element for compatibility with argv
+        file_list[0] = malloc (1);
+        file_list[0][0] = '\0';
+        char temp[200];
+        size_t length;
+        for (size_t i = 1 ; i < nfiles+1 ; i++)
+        {
+            fscanf (fr, "%s\n", temp);
```

Comment:
Stack overflow if path is longer than 200. Why not use `fgets()`? That solves the overflow and reports number of characters read so you can immediately raise an error if the path doesn't fit in the buffer.

Reply:
Good point. On my Linux desktop, gcc notes this as well, as a warning. I can revise this too.

Reply:
Should be addressed in #305.